packages/lesswrong/components/tagging/subforums/SubforumNotificationSettings.tsx

```diff
@@ -117,37 +115,24 @@ const SubforumNotificationSettings = ({
       subscriptionType, collectionName: "Tags", document: tag
     });
   }
-  const isSubscribed = getIsSubscribed();
-
-  const recordSubforumView = useRecordSubforumView({userId: currentUser._id, tagId: tag._id});
-  const userTagRel = results?.length ? results[0] : undefined;
-  
-  // This is to ensure the userTagRel exists, TODO refactor so this isn't necessary
-  useEffect(() => {
-    if (!loading && !userTagRel) {
-      void recordSubforumView().then(() => refetch())
-    }
-  }, [userTagRel, recordSubforumView, refetch, loading]);
-
-  // Don't show notification settings if the user is not subscribed to the tag
-  if (!currentUser || !currentUser.profileTagIds?.includes(tag._id)) return null;
-  if (!userTagRel) return null
+  const isSubscribedToPosts = getIsSubscribedToPosts();
 
   const filterSetting = filterSettings?.tags?.find(({tagId}) => tag._id === tagId);
   const isFrontpageSubscribed = filterSetting?.filterMode === "Subscribed";
 
-  const toggleIsFrontpageSubscribed = () =>
+  const toggleIsFrontpageSubscribed = () => {
     setTagFilter({
       tagId: tag._id,
       tagName: tag.name,
       filterMode: isFrontpageSubscribed ? 0 : "Subscribed",
     });
+  }
 
   const togglePostsSubscribed = async (e) => {
     try {
       e.preventDefault();
-      const subscriptionState = isSubscribed ? 'suppressed' : 'subscribed'
-      captureEvent("subscribeClicked", {state: subscriptionState})
+      const subscriptionState = isSubscribedToPosts ? 'suppressed' : 'subscribed'
+      captureEvent("subscribeClicked", {state: subscriptionState}) // TODO capture better events
```

Comment:
WTBU - is this a TODO for later?

Reply:
Ah yeah I did forget about this, I've made a ticket to come back to it now 👍